tests/jobs/test_scheduler_job.py

```diff
@@ -2822,6 +2824,7 @@ def test_verify_integrity_if_dag_changed(self):
         assert scheduler.dagbag.dags == {'test_verify_integrity_if_dag_changed': dag}
         assert len(scheduler.dagbag.dags.get("test_verify_integrity_if_dag_changed").tasks) == 1
 
+        SerializedDagModel.write_dag(dag=dag)
```

Comment:
This one doesn't seem like it would be needed as two lines later we write it.

Reply:
I think this one was releated to the scheduler processes still running in the background - see #14773 . I removed that line now with my tests.